CMakeLists.txt

```diff
@@ -389,8 +389,15 @@ set(OD_SOURCEFILES
     ${SRC}/utils/FrameRateLimiter.cpp
     ${SRC}/utils/Helper.cpp
     ${SRC}/utils/LogManager.cpp
-    ${SRC}/utils/LogManagerFile.cpp
-    ${SRC}/utils/LogManagerOgre.cpp
+    ${SRC}/utils/LogManager.h
+    ${SRC}/utils/LogMessageLevel.h
```

Comment:
The headers are not added to the generated vcxproj automatically, but must be added to the source list manually. I've removed the headers from the source list (as per the guideline), but I want to make a note that it's very annoying not to have the headers available in the solution explorer when working in Visual Studio.